### PR TITLE
Issue #1802 Add pause/resume operations to the pipe CLI

### PR DESCRIPTION
**Pull request by Wedds**
This PR is related to issue #1802

It brings `pause` and `resume` operations to the pipe CLI.

Besides, it is allowing specifying commands, which abortion will be indicated with a custom message.

### CODE REVIEW

File: pipe-cli/src/utilities/pipeline_run_operations.py [hunk 1]
```diff
@@ -290,6 +283,52 @@ def run(cls, pipeline, config, parameters, yes, run_params, instance_disk, insta
...
+            pipeline_name = cls.extract_pipeline_name(pipeline_run_model)
+            pipeline_version = pipeline_run_model.version
+            click.echo('Pausing RunID {} of "{}@{}"'.format(run_id, pipeline_name, pipeline_version))
+            if sync:
+                status = cls.get_pausing_pipeline_status(run_id)
+                if status == 'PAUSED':
+                    click.echo('RunID {} of "{}@{}" is paused'.format(run_id, pipeline_name, pipeline_version))
+                    sys.exit(1)
+                else:
+                    click.echo('Failed pausing RunID {} of "{}@{}"'.format(run_id, pipeline_name, pipeline_version))
```
Comment (ekazachkova) [inline pipe-cli/src/utilities/pipeline_run_operations.py]: Let's add `err=True` since this is the error message

File: pipe-cli/pipe.py [hunk 2]
```diff
@@ -698,6 +701,27 @@ def stop(run_id, yes):
...
+    """
+    PipelineRunOperations.pause(run_id, check_size, sync)
+
+
+@cli.command(name='resume')
+@click.argument('run-id', required=True, type=int)
+@click.option('-s', '--sync', is_flag=True, help=SYNC_FLAG_DESCRIPTION)
+@Config.validate_access_token
+def resume(run_id, sync):
+    """Resumes a running pipeline
```
Comment (ekazachkova) [inline pipe-cli/pipe.py]: Maybe `paused` would be more clear instead of `running`?

**Review by ekazachkova — CHANGES_REQUESTED** [review]

File: pipe-cli/src/utilities/pipeline_run_operations.py (hunk 1 above)
Comment (Wedds) [reply pipe-cli/src/utilities/pipeline_run_operations.py]: Done

File: pipe-cli/pipe.py (hunk 2 above)
Comment (Wedds) [reply pipe-cli/pipe.py]: Done

**Review by ekazachkova — APPROVED** [review]

**Pull request merged**